src/main/liberty/config/server.xml

```diff
@@ -2,21 +2,31 @@
 
   <featureManager>
       <feature>microprofile-1.3</feature>
+      <feature>usr:opentracingZipkin-0.30</feature>
   </featureManager>
 
   <httpEndpoint httpPort="${default.http.port}" httpsPort="${default.https.port}"
       id="defaultHttpEndpoint" host="*" />
 
   <!-- Automatically expand WAR files and EAR files -->
   <applicationManager autoExpand="true"/>
-
+    
+  <opentracingZipkin host="${env.zipkinHost}" port="${env.zipkinPort}"/>
+  
   <webApplication name="customer" location="${app.location}"/>
+           
+  <!--These values are defined in pom.xml-->
+    <mpJwt
+         id="${env.jwtid}"
+         jwksUri="${env.jwksUri}"
+         issuer="${env.jwksIssuer}">
+    </mpJwt>
+    
+    <administrator-role>
+      <user-access-id>user:${env.userRealm}/user</user-access-id>
+    </administrator-role>
 
-  <mpJwt
-     id="${env.jwtid}"
-     jwksUri="${env.jwksUri}"
-     issuer="${env.jwksIssuer}">
-  </mpJwt>
+  <logging traceSpecification="*=info:com.ibm.ws.security.*:com.ibm.ws.webcontainer.security.*=finest"/>
```

Comment:
You should remove the trace logging (The Metrics debugging before you merge in Zipkin)